tox.ini

```diff
@@ -1,18 +1,9 @@
 [tox]
-envlist = py38, py39, py3.10, py311, py312, pypy3
+envlist = py3, pypy3
```

Comment:
This changes how tox runs locally.  If I use pyenv or uv to install multiple Python versions on my machine, then the current form runs the tests on each of the listed Python versions that exist.  Does the proposed form do the same?

Reply:
You're right, it seems to only test the latest versions after this change. Do we need the full envlist, though? Anyone can specify the environments through the command line.
We can definitely use the full envlist if needed though. We have to chore-update the PyPI Python version classifiers anyways.